lib/models/guest.dart

```diff
@@ -0,0 +1,75 @@
+import 'package:json_annotation/json_annotation.dart';
+
+part 'guest.g.dart';
+
+@JsonSerializable()
```

Comment:
Skąd taki model?
Wg https://github.com/KoalicjaOtwartyKrakow/backend/blob/main/api.yaml jest nieco inny.

Reply:
Wzięłam stąd https://github.com/KoalicjaOtwartyKrakow/backend/blob/dev/utils/orm.py ale dopiero zauważyłam że mają #TODO: implementation does not match api.yaml "GuestCreate:" - więc dostosuję do modelu w api.yaml

Reply:
Pomimo zmian z [65eba2f](https://github.com/KoalicjaOtwartyKrakow/flutter-android/pull/36/commits/65eba2fff342ef9032f05deaf2a968d4af1507a6) dalej model się nie zgadza.

Tak powinien wyglądać wg swaggera:
![image](https://user-images.githubusercontent.com/3340954/159145111-4dcc161e-bbd0-40a8-97a2-1aa93365a1e7.png)

Reply:
Nazwy pól w jsonie (parametry `name` w adnotacjach JsonKey) dalej nie pasują.
Tak powinna wyglądać klasa jeśli to ma być spójne, pominąłem fabryki:

```dart
@JsonSerializable()
class Guest {
  final String fullName;
  final String email;
  final String phoneNumber;

  final int? peopleInGroup;
  final int? adultMaleCount;
  final int? adultFemaleCount;
  final List<ChildAge>? children;
  final bool? havePets;
  final String? petsDescription;
  final String? specialNeeds;
  final String? foodAllergies;
  final bool? meatFreeDiet;
  final bool? glutenFreeDiet;
  final bool? lactoseFreeDiet;
  final String? financeStatus;
  final String? howLongToStay;
  final String? desiredDestination;
  final PriorityStatus? priorityStatus;
  @JsonKey(
    fromJson: dateTimeFromString,
    toJson: nullableDateTimeToString,
  )
  final DateTime? priorityDate;

  VerificationStatus? verificationStatus;
  String? id;
  @JsonKey(
    fromJson: dateTimeFromString,
    toJson: nullableDateTimeToString,
  )
  DateTime? createdAt;
  @JsonKey(
    fromJson: dateTimeFromString,
    toJson: nullableDateTimeToString,
  )
  DateTime? updatedAt;

  Guest(
    this.fullName,
    this.email,
    this.phoneNumber,
    this.peopleInGroup,
    this.adultMaleCount,
    this.adultFemaleCount,
    this.children,
    this.havePets,
    this.petsDescription,
    this.specialNeeds,
    this.foodAllergies,
    this.meatFreeDiet,
    this.glutenFreeDiet,
    this.lactoseFreeDiet,
    this.financeStatus,
    this.howLongToStay,
    this.desiredDestination,
    this.priorityStatus,
    this.priorityDate,
  );
}

enum PriorityStatus {
  accommodation_not_needed,
  accommodation_found,
  en_route_ukraine,
  en_route_poland,
  in_krakow,
  in_crisis_point
}

enum VerificationStatus { created, verified, rejected }

@JsonSerializable()
class ChildAge {
  int? age;
}
```